src/hexdoc/patchouli/page/pages.py

```diff
@@ -47,13 +53,37 @@ class EmptyPage(Page, type="patchouli:empty", template_type="patchouli:page"):
 
 
 class EntityPage(PageWithText, type="patchouli:entity"):
+    _default_name: LocalizedStr = PrivateAttr()
+    _texture: PNGTexture = PrivateAttr()
+
     entity: Entity
     scale: float = 1
     offset: float = 0
     rotate: bool = True
     default_rotation: float = -45
     name: LocalizedStr | None = None
 
+    def get_name(self):
```

Comment:
I'd probably rename `name` to `name_field` and make this a `@property` called `name`, like for `SpotlightPage.title`.